src/lib/Event/Listener/LoginListener.php

```diff
@@ -43,11 +43,6 @@ class LoginListener
     private $logger;
 
     /**
-     * @param \Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface $authorizationChecker
-     * @param \Symfony\Component\HttpFoundation\Session\SessionInterface $session
-     * @param \EzSystems\EzRecommendationClient\Client\EzRecommendationClientInterface $client
-     * @param \eZ\Publish\API\Repository\UserService $userService
-     * @param \eZ\Publish\Core\MVC\ConfigResolverInterface $configResolver
      * @param \Psr\Log\LoggerInterface $logger
```

Comment:
all dependencies should be required (remove `?` from `?LoggerInterface`) NullLogger could be used if needed